moja-kuhinjica-frontend/src/components/modal/login/LoginModal.tsx

```diff
@@ -1,21 +1,23 @@
 import React, { useState } from 'react'
+import Image from 'next/image'
 import { useForm } from 'react-hook-form'
 import Modal from 'react-modal'
 import { FieldValues } from 'react-hook-form'
+import { useAppDispatch, useAppSelector } from '@/utils/hooks'
+import { userLogin } from '@/reduxStore/actions/userActions'
 import { FormInput } from '../../input/FormInput'
-import UserService from '@/service/User.service'
 import { ErrorLabel } from '@/components/label/ErrorLabel'
+import { Text } from '@/components/label/Text'
 import { bgModal } from '../../../constants/constants'
 import styles from './LoginModal.module.scss'
-import email from '../../../../public/static/assets/images/email.svg'
-import password from '../../../../public/static/assets/images/password.svg'
-import { Text } from '@/components/label/Text'
+import email from 'public/static/assets/images/email.svg'
+import password from 'public/static/assets/images/password.svg'
+import loadingIcon from 'public/static/assets/images/loading.png'
```

Comment:
You should add spinning icon to the Regular Button component and add it here

Reply:
Done.